### PR TITLE
Remove ghpages dependency and use new ghpages publishing

### PR DESCRIPTION
**Pull request by nylki**
[as of August 17](https://github.com/blog/2228-simpler-github-pages-publishing) its possible to use github pages from either `master` branch directly or a `/docs` directory inside `master`.

To make it easier for maintainers of a aframe-component and to reduce the amount of dependencies this merge-request proposes the move of `/examples` to `/docs` and changes the github repo settings to reflect that the ghpages now reside there.
The `gh-pages` branch is also going to be removed as well, as its would not be necessary anymore.

You can test the ghpages url for the forked repo: https://github.com/blog/2228-simpler-github-pages-publishing

PS: Not sure if the github repo settings are copied over in this PR? But setting them is straightforward.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,8 @@ Boilerplate for sharing [A-Frame](https://aframe.io) [components](https://aframe
 
 1. `npm install && npm run unboil` to rename and trim stuff. Don't forget to **unboil**.
 2. [Write your component](http://ngokevin.com/blog/aframe-component).
-3. Build examples (`npm run dev` to watch for changes to build example bundles).
+3. Build examples (`npm run dev` to watch for changes to build example bundles located in docs directory).
```
Comment (ngokevin) [inline README.md]: I'd backtick docs: "`docs/` directory"

File: README.md [hunk 2]
```diff
@@ -6,9 +6,8 @@ Boilerplate for sharing [A-Frame](https://aframe.io) [components](https://aframe
...
 4. `npm publish` and commit the `dist/` files.
-5. `npm run ghpages` to share with people.
 6. Share on [Slack](https://aframevr-slack.herokuapp.com/) and [awesome-aframe](https://github.com/aframevr/awesome-aframe)!
```
Comment (ngokevin) [inline README.md]: Decrease this to step 5 now

**Comment by ngokevin** [comment]
Thanks, I saw this announcement when it came out. I was initially excited, but now thinking about the tradeoffs. I am worried that this somehow makes things _more_ complicated:
- Have to know to go into the repository settings to change it. Rather than simply doing  `npm run ghpages` from where people already are working. 
- We lose control of when we want to simply commit work-in-progress commits, and we start having to work on dev branches.
- `examples/` is now named `docs/` which is misleading.

**Comment by nylki** [comment]
> Have to know to go into the repository settings to change it. Rather than simply doing npm run ghpages from where people already are working. 

As ghpages are a github feature already, I don't find it much of a problem, as long as the setting is copied over when forking the repo. This is not the case unfortunately, so I agree with you here if it stays like that.

Your other two points are valid. It would be nice to tell github any arbitrary directory to be used as github pages, not only `docs/`, I don't get why that is not an option :/
The work in progress commits are not that much of an issue from my point of view, but I can understand your position.

**Comment by ngokevin** [comment]
Thanks for understanding. Most people don't fork the boilerplate either to get started since they'd become their own projects rather than a simple downstream of the repo. The GitHub pages feature is nice such that people don't have to set up the ghpages script, but given we have it already set up, it's a smoother flow in general at the moment.

**Comment by nylki** [comment]
Alrighty, I'll close this one then :)

**Pull request closed without merging**